pkg/skoop/collector/manager/manager.go

```diff
@@ -77,6 +78,7 @@ func NewSimplePodCollectorManager(ctx *ctx.Context) (collector.Manager, error) {
 
 	return &simplePodCollectorManager{
 		image:                Config.SimplePodCollectorConfig.Image,
+		imagePullPolicy:      utils.ConvertToImagePullPolicy(Config.SimplePodCollectorConfig.ImagePullPolicy),
```

Comment:
Please add a validation to `Config.SimplePodCollectorConfig.ImagePullPolicy` and provide the possible values of it. It may be confusing that the `ImagePullPolicy` will fallback to `Always` if the user provide a wrong option.